src/components/Modal/Modal.tsx

```diff
@@ -8,32 +8,11 @@ import {
   Trigger as DialogTrigger,
   Content as DialogContent
 } from '@radix-ui/react-dialog';
-import { styled } from '@stitches/react';
 
 import { Button } from '../Button';
 
-// @TODO: use color variables
-
-const Overlay = styled(DialogOverlay, {
-  background: 'rgba(0 0 0 / 0.9)',
-  position: 'fixed',
-  top: 0,
-  left: 0,
-  right: 0,
-  bottom: 0,
-  display: 'grid',
-  placeItems: 'center',
-  overflowY: 'auto',
-  zIndex: 10000
-});
-
-const Content = styled(DialogContent, {
-  minWidth: 300,
-  background: '#0a0a0a',
-  padding: 30,
-  borderRadius: 16,
-  border: '1px solid #3b3b3b'
-});
```

Comment:
I really couldn't understand this styled component, but I am happy to see removing this 😄

Reply:
I was just testing out Radix when I made this component - they used Stitches in the example which I more or less copied, haha. I'm glad to purge it too!